pvactools/lib/filter.py

```diff
@@ -5,11 +5,12 @@
 pd.options.mode.chained_assignment = None
 
 class Filter:
-    def __init__(self, input_file, output_file, filter_criteria, int_filter_columns=[]):
+    def __init__(self, input_file, output_file, filter_criteria, int_filter_columns=[], filter_strategy=0):
```

Comment:
For easier code comprehension, I'm not a huge fan of using 0/1 here as the possible values for filter_strategy. I think using `AND`/`OR` or `any`/`all` would be more comprehensible.